aether-kernel/aether/kernel/api/mapping_validation.py

```diff
@@ -19,11 +19,15 @@
 import collections
 import json
 from django.utils.translation import ugettext as _
+import re
 
 from .entity_extractor import find_by_jsonpath
 
 import spavro
 
+ARRAY_ACCESSOR_REGEX = re.compile(
```

Comment:
You can take this constant from `entity_extrator`, do not redefine it twice :wink:

Reply:
:+1:  The regex is greedy and only splits on `[` & `]` so anything with <2 matches isn't an array accessor.